src/main/java/org/fundacion/jala/converter/controller/response/PaoPaoResponse.java

```diff
@@ -0,0 +1,25 @@
+package org.fundacion.jala.converter.controller.response;
+
+public class PaoPaoResponse {
+    private String status;
+
+    public PaoPaoResponse(final String initialStatus) {
+        this.status = initialStatus;
+    }
+
+    /**
+     * Gets the response status
+     * @return a String with the status
+     */
+    public String getStatus() {
+        return status;
+    }
+
+    /**
+     * Sets the response status.
+     * @param newStatus a String to change the status.
+     */
+    public void setStatus(final String newStatus) {
```

Comment:
the javadoc must have a space between the description and the return and param

Reply:
added it.